hawkbit-extension-multi-user/README.md

```diff
@@ -0,0 +1,25 @@
+# hawkBit multi user extension
+
+This extension allows you to statically configure multiple users through the application properties.
+When the extension is enabled but no users are configured, it will fall back to the standard security.user configuration.
+
+## Example configuration
+
+A list of permissions with description is [available from the hawkBit repository](https://github.com/eclipse/hawkbit/blob/master/hawkbit-security-core/src/main/java/org/eclipse/hawkbit/im/authentication/SpPermission.java).
+Additionally, if a single permission with the name *ALL* is configured, that user will be granted all available permissions.
+
+    multiuser.user[0].username=admin
+    multiuser.user[0].password=admin
```

Comment:
Since we migrated to Boot 2 you need to specify a prefix for password encoding (for every password in the list):     
```suggestion
    multiuser.user[0].password={noop}admin
```
For details have a look [here](https://spring.io/blog/2017/11/01/spring-security-5-0-0-rc1-released#password-storage-format)